vdev/src/commands/integration/mod.rs

```diff
@@ -1,5 +1,10 @@
 crate::cli_subcommands! {
-    "Manage integration test environments"
+    r#"Manage integration test environments
+
+These test setups are organized into a set of integrations, located in subdirectories
+`scripts/integration`.  For each integration, there is a matrix of environments, described in the
```

Comment:
We had briefly discussed moving these out of `scripts/`, is that still desired/planned later or is it not feasible?

Reply:
My plan is to get all of the integration tests working through vdev in a follow-up PR (work in progress), and then to refactor them and all the configuration files into a more suitable location.